driver/examples/plot_pcolormesh_cube.py

```diff
@@ -138,9 +157,14 @@ def gather_fortran_data_at_klevel(path: str, cn: int, var: str, klevel: int):
                 "You must specify the variable name (fortran_var) \
                     to be subtracted in Fortran data."
             )
-        fortran = gather_fortran_data_at_klevel(
-            args.fortran_data_path, args.size, args.fortran_var, args.zlevel
-        )
+        if args.fortran_from_wrapper:
+            fortran = gather_fortran_wrapper_at_klevel(
+                args.fortran_data_path, args.size, args.fortran_var, args.zlevel, 20
```

Comment:
We hard code this to 20 because we only have one set of wrapper data in this format for now. Will add longer runs later.